build_tools/travis/install.sh

```diff
@@ -53,23 +53,17 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            libgfortran mkl flake8 \
```

Comment:
I think at one point it was needed explicitly because dependencies were not correctly defined in the conda metadata. This has long been fixed.